Dockerfile-alpine.template

```diff
@@ -92,7 +92,8 @@ RUN apk add --no-cache --virtual .build-deps-yarn curl gnupg tar \
   && rm yarn-v$YARN_VERSION.tar.gz.asc yarn-v$YARN_VERSION.tar.gz \
   && apk del .build-deps-yarn \
   # smoke test
-  && yarn --version
+  && yarn --version \
+  && rm -fr /tmp/*
```

Comment:
Though this should work, consider the consistency and convention(?), maybe `rm -rf` instead of `rm -fr` would be better?

Reply:
done